src/__tests__/__snapshots__/country.test.js.snap

```diff
@@ -0,0 +1,278 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
Please add integration tests using React Testing Library.
You have added some snapshot tests. Please add unit tests for components and also for some pure functions in Redux code.

Reply:
Alright. I am working on it now